src/connectors/dynamodb.js

```diff
@@ -107,16 +107,18 @@ class Connector {
     );
   }
 
-  get(id, IndexName, pk) {
+  get(id, IndexName, pk, sk, skName) {
```

Comment:
Do you think this should just be an object so it doesnt matter the order of what's entered? This obviously isnt backwards compatible but not many people are using the 1.0 version yet.